data/actionsmap/yunohost.yml

```diff
@@ -1289,6 +1289,36 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
```

Comment:
I am not sure to understand why a cli/api exposition is needed for this.
Maybe I miss a use case ?